packages/fury-adapter-swagger/lib/parser.js

```diff
@@ -325,6 +325,22 @@ class Parser {
           });
         }
 
+        if (this.swagger.info.termsOfService) {
+          this.withPath('termsOfService', () => {
+            const { Link } = this.namespace.elements;
+            const link = new Link();
+
+            link.relation = 'terms-of-service';
+            link.href = this.swagger.info.termsOfService;
+
+            if (this.generateSourceMap) {
+              this.createSourceMap(this.api.meta.get('termsOfService'), this.path);
+            }
+
+            return this.api.push(link);
```

Comment:
The `link` element should go into the API Categories “links” meta property.

I think something like the following would work:

```suggestion
            return this.api.links.push(link);
```

Reply:
going to change